duecredit/tests/test_io.py

```diff
@@ -151,16 +150,17 @@ def myotherfunction(arg42):
     reference_numbers = []
     references = []
     for line in lines:
-        match_citation = re.search('(\[[0-9]\])$', line)
-        match_reference = re.search('^(\[[0-9]\])', line)
+        match_citation = re.search('\[(.+)\]$', line)
```

Comment:
they're separated by a comma if multiple citations -- maybe I'm not skilled enough in regexp to code it :-)